spec/automated_review/empty_database_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe "Empty Database" do
+  let(:connection) { ApplicationRecord.connection }
```

Comment:
I know we use `ActiveRecord::Base` in the actual migrations. Should we use it here too?